source/main/index.js

```diff
@@ -123,20 +121,8 @@ app.on('open-file', (e, p) => {
  * else. It is necessary to wait for the ready event, because prior, some APIs
  * may not work correctly.
  */
-app.whenReady().then(() => {
+app.on('ready', () => {
```

Comment:
The whenReady() method is (apparently) the preferred way: https://github.com/electron/electron/pull/21972

Reply:
I already changed these things in upstream, so a git pull would resolve this in a better way, I'd suggest!

Reply:
I have rebased my change on the latest develop branch.

Reply:
Perfect! Short question, as I don't run into that problem: How difficult is rebasing and pulling in the latest changes? Is it easy, or bothersome for you?

Reply:
It is easy until there is a conflict, e.g. a file is modified on `develop` and my change also tries to change that file. At that point, it will require the same conflict resolution process as merging `develop` into my branch, but this keeps things cleaner.

Reply:
Yes, this makes sense. But in this regard, now we know that all branches are up to date with develop, which'll break fewer things, hopefully! Thanks for the info :)